docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -29,17 +30,47 @@ Once the destination index is built, the {dfanalytics-job} task calls the {es}
 ==== Loading data
 
 After the reindexing is finished, the job fetches the needed data from the 
-destination index. It converts the data into the format that the analysis process 
-expects, then sends it to the analysis process.
+destination index. It converts the data into the format that the analysis 
+process expects, then sends it to the analysis process.
 
 
 [discrete]
 ==== Analyzing
 
 When all the required data is loaded, the analyzing phase begins. The exact 
 process always depends on the type of {dfanalytics} – for example, 
-{classification} or {oldetection} – that is executed. This is the phase when the 
-job generates a {ml} model for analyzing the dataset.
+{classification} or {oldetection} – that is executed, you can find the 
+description of these processes below ordered by analysis type. The analyzing 
+phase is not reported in the `phase` property of the 
+{ref}/get-dfanalytics-stats.html[Get {dfanalytics-jobs} statistics API], the 
+specific sub-phases are reported instead.
+In this phase, the job generates a {ml} model for analyzing the dataset.
+
+
+[discrete]
+===== {oldetection-cap}
+
+{oldetection-cap} has one sub-phase during analyzing that called 
+`computing_outliers`. This is the phase when the {oldetection} analysis 
+identifies outliers in the loaded data.
+
+
+[discrete]
+===== {regression-cap} and {classanalysis}
+
+{regression-cap} and {classification} share the same set of sub-phases. There 
+are four of them:
+
+* `feature_selection`: In this phase, the process iterates through the available 
```

Comment:
If these phases occur in a specific order, it might make sense to use a numbered list here. I'd also suggest making each bullet point less verbose. Alternatively, you could remove the list and discuss each phase in order.

```suggestion
. `feature_selection`: Identifies which fields are significant for the analysis 
```